src/content/roadmap/future-proofing.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Nitpick but noticing some of these pages are within directories (e.g `/danksharding/index.md`) but some are not (e.g. `/future-proofing.md`) - can we please be consistent with our approach?

Reply:
To be clear here:

- The only `.md` file within the roadmap directory should be `index.md`
- Every subpage should be in a folder, and the markdown page should also be named `index.md`

Reply:
ah, yea. fixed. Thanks!